google/cloud/spanner/value.cc

```diff
@@ -413,6 +426,12 @@ google::protobuf::Value Value::MakeValueProto(PgNumeric n) {
   return v;
 }
 
+google::protobuf::Value Value::MakeValueProto(PgOid n) {
+  google::protobuf::Value v;
+  v.set_string_value(std::string(std::move(n)));
```

Comment:
Here is where we would finally serialize, using...
```
  v.set_string_value(std::to_string(std::uint64_t(n)));
```

Reply:
Done